Makefile

```
@@ -27,40 +27,24 @@ PYTEST_ARGS ?=
 BUILD_IMAGES ?= 1
 
 ifdef KEEP_TEST_CONTAINER
-  DOCKER_RM_CONTAINER =
+  CONTAINER_RM =
 else
-  DOCKER_RM_CONTAINER = --rm
+  CONTAINER_RM = --rm
 endif
 
-# Let the user specify DOCKER at the CLI, otherwise try to autodetect a working podman or docker
-ifndef DOCKER
-  DOCKER := $(shell podman run --rm alpine echo podman 2> /dev/null)
-  ifndef DOCKER
-    DOCKER := $(shell docker run --rm alpine echo docker 2> /dev/null)
-    ifndef DOCKER
-      DUMMY := $(warning Many of the make targets require a working podman or docker.  Please install one of those and check that `podman run alpine echo hello` or `docker run alpine echo hello` work)
-    endif
-  endif
+# Let the user specify PODMAN at the CLI, otherwise try to autodetect a working podman
+ifndef PODMAN
+  PODMAN := $(shell podman run --rm alpine echo podman 2> /dev/null)
 endif
 
-ifdef DOCKER
-  ifeq ($(DOCKER), podman)
-    DOCKER_TEST_WARNING := "*** convert2rhel directory will be read-only while tests are executing ***"
-    DOCKER_CLEANUP := podman unshare chown -R 0:0 $(shell pwd)
-  else
-    # Docker
-    DOCKER_TEST_WARNING := -n
-    DOCKER_CLEANUP := echo -n
-  endif
-else
-  # No docker found
-  DOCKER_TEST_WARNING ?= -n
-  DOCKER_CLEANUP ?= echo -n
+ifdef PODMAN
+	CONTAINER_TEST_WARNING := "*** convert2rhel directory will be read-only while tests are executing ***"
+	CONTAINER_CLEANUP := podman unshare chown -R 0:0 $(shell pwd)
 endif
```

I think the indentation here should be like this, right?

```suggestion
ifdef PODMAN
   CONTAINER_TEST_WARNING := "*** convert2rhel directory will be read-only while tests are executing ***"
   CONTAINER_CLEANUP := podman unshare chown -R 0:0 $(shell pwd)
endif
```

---

Good catch